modules/installation-and-upgrade/pages/network-requirements.adoc

```diff
@@ -37,6 +28,28 @@ You also need to ensure that reverse lookups are correctly configured.
 
 For more information about setting up a DNS server, see {sles-base-os-documentation}/cha-dns.html.
 
+
+== Reenable router advertisements
+
+When the {productname} is installed using [command]``mgradm install podman`` or [command]``mgrpxy install podman``, it sets up Podman which enables IPv4 and IPv6 forwarding.
+This is needed for communication from the outside of the container.
+
+However, if your system previously had [literal]``/proc/sys/net/ipv6/conf/eth0/accept_ra`` set to [literal]``1``, it will stop using router advertisments.
```

Comment:
```suggestion
However, if your system previously had [literal]``/proc/sys/net/ipv6/conf/eth0/accept_ra`` set to [literal]``1``, it will stop using router advertisements.
```